lemur/factory.py

```diff
@@ -136,7 +142,22 @@ def configure_extensions(app):
     principal.init_app(app)
     smtp_mail.init_app(app)
     metrics.init_app(app)
-    sentry.init_app(app)
+
+    # the legacy Raven[flask] relied on SENTRY_CONFIG
+    if app.config.get("SENTRY_DSN", None) or app.config.get("SENTRY_CONFIG", None):
+        # priority given to SENTRY_DSN
+        sentry_dns = app.config.get("SENTRY_DSN", None) or app.config["SENTRY_CONFIG"]['dsn']
```

Comment:
Should `sentry_dns` be `sentry_dsn`?

Reply:
good catch, Sentry's Data Source Name (DSN) get's me all the time confused.